### PR TITLE
Removes the traitor assistant toolbox

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
This PR removes the funny traitor assistant toolbox that scales damage based on how many TC are inside of it. For those of you who don't know, this toolbox:

- Costs 2 telecrystals
- Is entirely inconspicuous other than the insane damage
- Deals 19 brute by itself without any crystals (50% chance to dislocate a limb on hit) (for a 2TC item!!)
- Deals 37 brute with the other 18 telecrystals (50% chance to cause an outright hairline fracture on the first hit, and a **20% chance to cause an outright compound fracture on the first hit**)
- Leaves the bodies of victims crippled with broken bones requiring long surgeries to repair even after they're revived
- Also gives you a full set of tools and insulated gloves

Every time I've seen this dumb toolbox in use, the round sucked. I've seen three traitors go together and have a toolbox that did 77 brute, and that round sucked too. Sometimes broken and wildly unbalanced things can be funny, but this is not one of them.

## Why It's Good For The Game
This item sucks and probably only still exists because anything relating to assistants and toolboxes (especially old toolboxes) is considered SOUL, but I was also the one who added sacrificing souls to toolboxes, so maybe I'm the right person for the job

## Changelog
:cl: Ryll/Shaps
del: Removed the assistant traitor toolbox
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/toolbox.dm [hunk 1]
```diff
@@ -97,39 +97,6 @@
 /obj/item/storage/toolbox/mechanical/old/heirloom/PopulateContents()
 	return
 
-/obj/item/storage/toolbox/mechanical/old/clean // the assistant traitor toolbox, damage scales with TC inside
-	name = "toolbox"
-	desc = "An old, blue toolbox, it looks robust."
-	icon_state = "oldtoolboxclean"
```
Comment (LemonInTheDark) [inline code/game/objects/items/storage/toolbox.dm:103]: keeping the icon state?

**Comment by Krysonism** [comment]
The toolbox is not OP.

Melee weapons by in large suck. 

Especially when compared to the non-bulky, 0 TC, 60 damage and armour piercing batong.

**Comment by rellyreri** [comment]
why dont you nerf it instead of removing it entirely

**Comment by Yenwodyah** [comment]
* Toolboxes are absolutely not inconspicuous. The only reason anyone ever carries a toolbox around is for robusting (since toolbelts or your backpack are so much better for actually carrying tools around). 
If you see an assistant carrying a toolbox around and don't immediately realize that they're up to no good, that's on you.

* 19 brute is not that much, a wielded spear deals more than that.

* Consider that for the same price as your 37-damage toolbox that you'll lose as soon as someone hits you with a stunbaton or shoves you over, you could also buy a desword that deals 34 damage, dismembers, has a 75% block chance, and fits in your pocket  AND buy a 15-damage toolbox with better syndie tools and combat gloves AND have 1 tc left over. 

Any problem that exists is with the terrible, overly-punishing wound system, not the toolbox.

**Comment by CRITAWAKETS** [comment]
> * Is entirely inconspicuous other than the insane damage
its the only item in the game with that sprite

**Comment by Ryll-Ryll** [comment]
> Any problem that exists is with the terrible, overly-punishing wound system, not the toolbox.

I think the problem is with the weapon that does 4x the damage of the average blunt weapon

> > * Is entirely inconspicuous other than the insane damage
> 
> its the only item in the game with that sprite

it looks like a toolbox, not like something that would crit you in 3 hits

**Comment by zxaber** [comment]
What if the toolbox just didn't apply wounds? Is that even possible?

**Comment by Ryll-Ryll** [comment]
It's easy to make it unable to deal wounds but the item itself is still trash

**Comment by SinfulBliss** [comment]
Never seen this happen but if it did happen it'd be a memorable round. As long as it's rare it ain't broken (and it's hard cause you need good trust+coordination among traitors).

**Comment by Yenwodyah** [comment]
>I think the problem is with the weapon that does 4x the damage of the average blunt weapon
If you're spending 20 TC on it it's not supposed to be average, it's supposed to be incredibly deadly.
For comparison the powerfist does 60 damage, is 6 TC, and can fit in your backpack.
(Oh look at that, the powerfist has wounding turned off. It's almost as if there's an issue with wound scaling for high damage blunt weapons)

**Comment by LemonInTheDark** [comment]
This is what the 👍 emoji was created for

**Comment by tralezab** [comment]
We have a zillion uplink items (I had to iterate through them all on biddle traitors pr) and let me say this one is so brainless and dumb. I'm in favor of any kind of shortening of the traitor uplink selection but especially this item

**Comment by LemonInTheDark** [comment]
I respect you for doing this. Making value judgements about this sort of thing is tough, especially + backlash

**Comment by ATH1909** [comment]
> * 19 brute is not that much, a wielded spear deals more than that.

a wielded spear has a force of 18, actually. your point is still valid, though.

**Comment by Mothblocks** [comment]
the budget limit on the uplink will be real

File: code/game/objects/items/storage/toolbox.dm (hunk 1 above)
Comment (Mothblocks) [reply code/game/objects/items/storage/toolbox.dm:103]: yeah, should remove this

**Comment by Fritzlschnitzel** [comment]
amazing an item that is slighty better than the stunbaton. Only after three traitors team up and put all their tc into one item. sorry bucko but this is a salt pr nothing wrong with it or if there would be anything wrong with it are all the damaging items a problem?

File: code/game/objects/items/storage/toolbox.dm (hunk 1 above)
Comment (Fikou) [reply code/game/objects/items/storage/toolbox.dm:103]: im pretty sure its used in a few places or maps

**Comment by Fikou** [comment]
its fun

File: _maps/RandomRuins/IceRuins/icemoon_underground_lavaland.dmm [hunk 2]
```diff
@@ -53,12 +53,6 @@
 "T" = (
 /turf/open/floor/cult,
 /area/icemoon/underground)
-"V" = (
```
Comment (NamelessFairy) [inline _maps/RandomRuins/IceRuins/icemoon_underground_lavaland.dmm:56]: Didn't even realize that this toolbox was the traitor one but any reason in particular why its being removed all together and not replaced with an alternate toolbox

**Comment by maxymax13** [comment]
oh nooo if you get 60tc you can get something MODERATELY POWERFUL oh nooooooooo!

**Comment by Fikou** [comment]
if someone doesnt power the traitor toolbox, its just a bulky energy dagger that gives u insuls
if they power the traitor toolbox, they leave themselves defenseless and you can kill them by slipping them once or using ranged/meme weapons
the moment someone shiftclicks you they know you have the traitor toolbox, and can call on radio to let the entire station know
the only moment it gets really strong is when you have multiple traitors teaming up (a good thing) and multiple traitors teaming up might as well do zombies or other shit that puts the ground to hell

File: code/game/objects/items/storage/toolbox.dm (hunk 1 above)
Comment (Rohesie) [reply code/game/objects/items/storage/toolbox.dm:103]: It's not, it's safe to remove.

**Comment by interestingusernam3** [comment]
>It's easy to make it unable to deal wounds but the item itself is still trash

ided

**Comment by wesoda25** [comment]
It's the umpteenth iteration of an item whose sole purpose is murder, except this one isn't even that interesting. 👋

**Comment by cacogen** [comment]
You should just nerf it instead of making a PR that's probably justified but just makes it look like you died and is abrasive because it's taking it upon yourself to delete something people like from the game without even making a good faith effort to fix the problems you see with it.

I don't see how wounds are interesting, either. It's time to hope medbay exist, know what they're doing and are doing their jobs so you can lie in a stasis bed for five minutes if you're lucky.

**Comment by nianjiilical** [comment]
why not just rebalance it so that it needs more than 20tc to become unusually powerful

i mean this is literally an ided pr there has to be a better way

**Comment by Technoturnovers** [comment]
get fucking real, it's a melee weapon, and if you're gonna effectively spend 20tc on something, it better be really, really good. and as @Yenwodyah said, without any TC in it, it's not even as good as a wielded spear, something that you can make for FREE.

**Comment by Farquaar** [comment]
> uplink



> 
> 
> We have a zillion uplink items (I had to iterate through them all on biddle traitors pr) and let me say this one is so brainless and dumb. I'm in favor of any kind of shortening of the traitor uplink selection but especially this item

If your goal is to shorten the uplink list, how would removing an role-restricted item help with that? The role-restricted tab is one of the shortest in the whole system.

**Comment by zxaber** [comment]
The toolbox is a neat concept because it's a traitor melee weapon that deals more damage the less you need the uplink. It's super powerful but loses strength every time you have to get another traitor item.

Capping at 18 is probably a good idea (since the above is bypassed if you just get more TC from another traitor), and the free tools and gloves should get thrown out (get them through other in-game methods or lower your toolbox damage and buy the tool kit if you're lazy).

It's certainly more interesting than pocket-sized sword that also provides ranged weapon resistances.

**Comment by 81Denton** [comment]
I'm gonna agree with @Yenwodyah and @Fikou on this one. 77 is a ton of brute damage, but if three traitors team up they can derail rounds and murder people way more easily than by pooling all their TC into a single melee weapon. Even just two traitors teaming up to buy Romerol will do that.

Compared to other melee weapons, the TC/damage value is lower compared to the esword/desword, and you don't get the concealability or block/reflect chance that those offer. I'd argue that a traitor with a desword and EMP kit/noslips is more dangerous than one with a box that deals 3 more damage.

Regarding wounds, it's my opinion that strong weapons easily crippling players through fractures is more of an issue of wounds themselves being debilitating and time-consuming to fix. Hell, even the powerfist has `wound_bonus = CANT_WOUND` set so it doesn't turn bones into mush on each hit.

**Comment by Mothblocks** [comment]
i was too late

**Comment by tralezab** [comment]
When I said boomertainers will defend this to my friend I didn't mean for them to actually defend this wtf

**Comment by 81Denton** [comment]
> When I said boomertainers will defend this to my friend I didn't mean for them to actually defend this wtf

aahh

**Pull request closed without merging**